src/accelerate/accelerator.py

```diff
@@ -1986,6 +1986,10 @@ def unscale_gradients(self, optimizer=None):
             for opt in optimizer:
                 while isinstance(opt, AcceleratedOptimizer):
                     opt = opt.optimizer
+                # Reduce gradients first for XLA
+                if is_tpu_available():
+                    gradients = xm._fetch_gradients(opt)
+                    xm.all_reduce("sum", gradients, scale=1.0 / xm.xrt_world_size())
```

Comment:
```suggestion
                # Reduce gradients first for XLA
                if self.distributed_type == DistributedType.TPU:
                    gradients = xm._fetch_gradients(opt)
                    self.reduce(gradients, scale=1.0 / self.num_processes)
```

We should then update `accelerator.reduce to take in a `scale`, as well as `accelerate.utils.operations.reduce`, noting that `scale` is only valid on XLA.